README.md

```diff
@@ -28,42 +30,182 @@ path/to/discern/data/
 Please follow this notebook naming convention for exploratory notebooks in the shared Switchdrive folder: 
 `<number>_<initials>_<short_description>.ipynb`. 
 
-### Example Usage
+## Example Usage
 
+### If all you care about is loading clean data to be on your merry way...
 ```python
 # IPython magics for auto-reloading code changes to the library
 %load_ext autoreload
 %autoreload 2
 
 import autodiscern as ad
+
+# See "Note on Data" above for what to pass here
+dm = ad.DataManager("path/to/discern/data")
+
+# Load up a pickled data dictionary.
+# automatically loads the file with the most recent timestamp
+# To load a specific file, use
+#   dm.load_transformed_data('filename')
+transformed_data = dm.load_most_recent_transformed_data()
+
+# transformed data is a dictionary in the format {id: data_dict}.
+# Each data dict represents a snippet of text, and contains keys with information about that text.
+# Here is an example of the data structure:
+{
+    '205-3': {
```

Comment:
I think this example outdated (i.e. the content part does not correspond to the current dictionary of article 205 -- compare it to the latest pickled data)

Reply:
I did not intend for this example to match the true output for 205-3- I intentionally crafted a fictitious example here that shows all of the fields being used. I don't think there exists a single sentence that has tags, links, and metamap concepts all in the same sentence. Do you think I should change the '205-3' key to something else to make it more clear this is a fictitious example?

Reply:
Oh, I see now. Yes, I think using fictitious id would be better.